### PR TITLE
NDI-59: Fix style glitches for event and campaign page

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
# Description
This PR
 - Fixes style for new campaign page
 - Fixes dots for Manage event pages
 - Fixes width for textarea as common to adapt width of vanilla civicrm

# Screenshots
**Before**
after-1
**After**
after-1

**Before**
before-2
**After**
after-2

# Technical Details
Below code in `scss/civicrm/common/_forms.scss` is used to make width as per Vanilla CiviCRM.
```scss
&.big {
  width: 35em;
}

&.huge {
  width: 45em;
}
```

# Comments
- Stylelist tested
- BackstopJS tested

### CODE REVIEW

File: scss/civicrm/common/_forms.scss [hunk 1]
```diff
@@ -113,6 +113,14 @@
       @include placeholder;
       box-sizing: border-box;
       width: unset;
+
+      &.big {
+        width: 35em;
+      }
+
+      &.huge {
```
Comment (AkA84) [inline scss/civicrm/common/_forms.scss]: if these are the same widths in vanilla civicrm, why do we have to define them? Are we overriding them somewhere?

Comment (mukeshcompucorp) [reply scss/civicrm/common/_forms.scss]: Yes, we extend `.from-control` from bootstrap which makes it  `width: 100%`.

**Comment by jamienovick** [comment]
@mukeshcompucorp can we align the ellipses a little bit better to the menu title for the below:
https://user-images.githubusercontent.com/26058635/37517961-3999ee10-2939-11e8-9480-11068e1079d9.png

Also I think the manage my personal campaign page is a public CiviCRM page - i.e. for people who have created their fundraising page, so we should have minimal styles on this I think. It may be good to check in the mattermost dev room and get people's views on this.

Thanks

**Comment by mukeshcompucorp** [comment]
@jamienovick Sorry, I'm not getting you about better title. Can you elaborate more on that?

For the public pages, we are already discussing that with @colemanw in our collaboration room, This is ticket like NDI-71 so, we will not be applying style to this page at all once, NDI-71 is done.

The idea is to not load our shoreditch css in such pages as we discussed few days back.

**Comment by jamienovick** [comment]
Sorry yes bad comment - the ellipses - can we align them better to the label for the menu? i.e.: 
image
They just look a little bit low compare to the text?

**Comment by mukeshcompucorp** [comment]
Ok that one is fixed on live screen already 
image

**Comment by jamienovick** [comment]
Much better thanks!

**Pull request merged**